tests/integration/targets/vmware_ops_esxi_maintenance_mode_test/vars/main.yml

```diff
@@ -16,7 +16,7 @@ provision_virtual_esxi_folder: ""
 provision_virtual_esxi_resource_pool: "{{ resource_pool_name }}"
 provision_virtual_esxi_datastore_iso_path: "{{ custom_esxi_8_iso_path }}"
 provision_virtual_esxi_vms:
-  - name: ci-vcenter-maintenance-test
+  - name: "{{ tiny_prefix }}-vm-esxi_maintenance-test"
 provision_virtual_esxi_networks:
     - name: "{{ test_network_name }}"
```

Comment:
I guess we should add `tiny_prefix` to `test_network_name` in group_vars.yml at https://github.com/redhat-cop/cloud.vmware_ops/blob/main/tests/integration/targets/group_vars.yml#L13

Reply:
since `TestNetwork` is permanent, no need to add `tiny_prefix`